source/_docs/blueprint/selectors.markdown

```diff
@@ -259,6 +265,20 @@ This selector does not have any other options; therefore, it only has its key.
 assist_pipeline:
 ```
 
+## Backup location selector
+
+This can only be used on an installation with a Supervisor. For installations
```

Comment:
```suggestion
This can only be used on an installation with a Supervisor. It cannot be used on installations of type Home Assistant Core. For installations
```
I think this would make it more clear. (My first thought was that this applies to only Home Assistant Supervised installations)

Reply:
This didn't really read right to me. The full paragraph would've been this:
> This can only be used on an installation with a Supervisor. It cannot be used on installations of type Home Assistant Core. For installations that do not have that, an error will be displayed.

It wasn't clear to me what "that" was in the last sentence with the one in between. I tweaked it a little bit to specifically mention which installation types work and which show an error instead. Let me know what you think.

Reply:
Perfect!  Thank you for fixing it. I wasn't actually sure which ones you meant with "Supervisor". So I went out on a limb and hoped you would specify if necessary.